### ISSUE TITLE
Facet Navigation and Screen Reader(s)

### ISSUE THREAD

**Issue by mtribone** [projectblacklight/blacklight#1905]
When testing Blacklight 7 with the available online demo (http://demo.projectblacklight.org/) our users with the accessibility group discovered some unexpected behavior when using the facets to filter search queries.

1.  The caret or disclosure triangle that is added using `::after` for the facet header is read aloud using Jaws. It is recommended to hide the caret from screen readers since the facet header is linked and performs the same actions with more relevant information.
1. Opening and closing facets has some unexpected behavior. When a facet is "closed" the focus stays within the context of the items within the facet and a user can still navigate within the items even when the facet has been closed. The items inside of the closed facet can still be read by the screen reader. Depending on which screen reader is used (JAWS or NVDA) closing a facet moves the context to the top of the list of facets instead of remaining on the header of the facet that was just closed. This can be confusing for users.
1. Some unnecessary chatter with special characters in facets and search. Make sure that special characters are escaped.

**Comment by barmintor** [comment]
Is "the accessibility group" here an entity that we can work with to address these issues?

**Comment by mtribone** [comment]
@barmintor We'd have to fix it based on their recommendations and then re-test with them. I'm cycling between projects, so it would be at least a month before I can fix it up and have them use it to provide feedback.

**Comment by barmintor** [comment]
@mtribone ok, I can understand that. In light of it, and because this doesn't appear to affect backwards compatibility, I'm going to move this to the 7.x milestone and the 7.0 backlog (on the off chance that resources become available).

**Comment by barmintor** [comment]
Oh, I see my work here is done. By me. Thats embarrassing.

**Comment by mtribone** [comment]
I didn't put these upstream issues in our CHO issues because of the previous accessibility work. I couldn't join that effort because of other commitments, so they've been added for the next round of CHO.

**Comment by VanessaVenti** [comment]
@mtribone in testing Blacklight 8 (on our Spotlight instance) here at Harvard we discovered the following issue with facets. Should I add this info to the description on this ticket or create a new ticket?

- **VoiceOver**: When expanded, the first item (e.g., creator, date) reads the name of the filter, not the count associated with the filter. Each filter should include the number of items related to the filter when reading out to screenreaders. 
- **WCAG Criteria**: 1.1.1 Non-text Content
- **Audit Method**: Screenreader testing
- **Action**: Ensure all items and their respective counts are read correctly
